docs/detections/query-alert-indices.asciidoc

```diff
@@ -0,0 +1,23 @@
+[[query-alert-indices]]
+== Query alert indices
+This page explains how you should query alert indices, for example when building rule queries, custom dashboards, or visualizations. For more information about alert event field definitions, review the [Alert schema].
+
+[discrete]
+=== Alert indices
+Alert events are stored in hidden {es} indices. The naming conventions for these indices and their aliases differ depending on which version of {stack} you're running:
+
+* *8.0.0 and later:* `.internal.alerts-security.alerts-<space-id>-NNNNNN`, where `NNNNNN` is a number that increases over time, starting from 000001.
+
+* *Before 8.0.0:* `.siem-signals-<space-id>-NNNNNN`, where `NNNNNN` is a number that increases over time, starting from 000001.
```

Comment:
I would consider moving this example of how **not** to query the alerts indices down below the recommended method. If a user stops reading the page after this section, they have all the information needed to query the alert indices incorrectly and haven't reached any text warning them that these are actually not the recommended ways.

It might also be helpful to include a warning within this section that this section is provided only for context and to provide a concrete example of how correct and incorrect usages differ.

Reply:
Great feedback, thanks.